newdle/client/src/components/creation/CreationSuccessPage.js

```diff
@@ -38,7 +38,26 @@ export default function CreationSuccessPage() {
             invite:
           </p>
         )}
-        <div className={styles['newdle-link']}>{newdle.url}</div>
+        <Input
+          className={styles['newdle-link']}
+          readOnly
+          value={newdle.url}
+          action={
+            <Popup
+              content="Copied!"
+              on="click"
+              position="top center"
+              inverted
+              trigger={
+                <Button
```

Comment:
The asynchronous Clipboard API is not supported on Edge, so I would probably hide this button in that case. We can just check the truthiness of `navigator.clipboard`, I believe. Anyway, it would be nice if we test it with Edge before we merge.

Reply:
That's weird, generally localhost should be exempt from "https-only" restrictions. If not I'd consider it a bug in the browser...

Reply:
[Found on stackoverflow](https://stackoverflow.com/a/51823007)

Reply:
oops, I guess I misread your last comment as "doesn't work on localhost".

Reply:
`action={navigator.clipboard && <...>}` should be fine, we did the same in the indico PR linked in a commend of the original issue